packages/oasis-actions/src/strategies/ajna/earn/open.ts

```diff
@@ -61,42 +60,34 @@ export async function open(
   )
 
   const priceIndex = await poolInfo
-    .priceToIndex(ethers.utils.parseUnits(args.price.toString(), 18).toString())
+    .priceToIndex(args.price.shiftedBy(18).toString())
     .then((res: any) => res.toString())
     .then((res: string) => new BigNumber(res))
 
-  const data = await ajnaProxyActions.interface.encodeFunctionData(
+  const data = ajnaProxyActions.interface.encodeFunctionData(
     args.isStakingNft ? 'openEarnPositionNft' : 'openEarnPosition',
     [
       args.poolAddress,
       ethers.utils.parseUnits(args.quoteAmount.toString(), args.quoteTokenPrecision).toString(),
-      ethers.utils.parseUnits(args.price.toString(), 3).toString(),
+      args.price.shiftedBy(18).toString(),
```

Comment:
are you sure it's accepting price with precision 18?

Reply:
yes, we needed this precision and we adjusted AjnaProxyActions in order to handle it